controllers/volumereplicationgroup_controller.go

```diff
@@ -981,7 +981,6 @@ func (v *VRGInstance) relocate(result *ctrl.Result, s3StoreAccessors []s3StoreAc
 	); clusterDataProtected != nil && (clusterDataProtected.Status != metav1.ConditionTrue ||
 		clusterDataProtected.ObservedGeneration != vrg.Generation) {
 		v.kubeObjectsProtectSecondary(result, s3StoreAccessors)
-		v.vrgObjectProtect(result, s3StoreAccessors)
```

Comment:
Why is this call removed?

Reply:
`kubeObjectsProtectSecondary` now does the VRG upload.  There are two reasons for uploading the VRG to S3:
- `spec` and `metadata` are needed for `ProtectedVRGList` API
- `status` is needed for the pointer to the latest Kube objects backup: 0 or 1

This PR separates the two use cases where `vrgObjectProtect` is now only used for the former and `kubeObjectProtect*` the latter.